index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Poppins" />
+
+    
+    <link rel="stylesheet" href="./styles.css">
+    <title>Rafael Echart Portafolio</title>
+</head>
+<body>
+    
+
+        <header class="header">
+            <span class="logo">Rafael Echart</span>
+            <input class="menu-btn" type="checkbox" id="menu-btn" />
+            <label class="menu-icon" for="menu-btn"><span class="navicon"></span></label>
+            <ul class="menu">
+                <li><a href="#portofolio">Portafolio</a></li>
+                <li><a href="#about">About</a></li>
+                <li><a href="#contact">Contact</a></li>
+            </ul>
+        </header>
+
+
+        <div class="background center-title">
+
+            <div class="welcome-items">
+                <h1>
+                    I'm Rafael<br/>
+                    Glad to see you!
+                </h1>
+    
+                <h5>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</h5>
+                <h4 class="subtitle-color">LET'S CONNECT</h4>
+                <div class="socialmedia-align">
+                    <img src="./assets/twitter.png" alt="twitter_logo">
+                    <img src="./assets/github.png" alt="github_logo">
+                    <img src="./assets/Linkedin.png" alt="linkedin_logo">
+                </div>
```

Comment:
- [ ] The list of icons uses a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag)